week-3/D-array-filter/exercise.js

```diff
@@ -8,7 +8,11 @@
 
 var pairsByIndexRaw = [[0, 3], [1, 2], [2, 1], null, [1], false, "whoops"];
 
-var pairsByIndex; // Complete this statement
+var pairsByIndex = pairsByIndexRaw.filter(function(aIndex) {
+  if (aIndex !== []) {
```

Comment:
This always returns true. You want to use `Array.isArray(item)`. Additionally, you want to check the `length == 2`